### PR TITLE
Translate curl_setopt constants

### PR DESCRIPTION
**Pull request by Fan2Shrek**
This is big file here.

I hope I haven't missed anything ^^'

### CODE REVIEW

File: reference/curl/constants_curl_setopt.xml [hunk 1]
```diff
@@ -0,0 +1,3094 @@
...
+  <title><function>curl-setopt</function></title>
+  <varlistentry xml:id="constant.curlopt-abstract-unix-socket">
+   <term>
+    <constant>CURLOPT_ABSTRACT_UNIX_SOCKET</constant>
+    (<type>int</type>)
+   </term>
+   <listitem>
+    <para>
+     Active l'utilisation d'un socket Unix abstrait à la place
+     d'établir une connexion TCP à un hôte et définitle chemin donné
```
Comment (devnexen) [inline reference/curl/constants_curl_setopt.xml:14]: space forgotten after `définit`

File: reference/curl/constants_curl_setopt.xml [hunk 2]
```diff
@@ -0,0 +1,3094 @@
...
+     en <type>string</type>. Cette option partage les mêmes sémantiques
+     que <constant>CURLOPT_UNIX_SOCKET_PATH</constant>. Ces deux options
+     partagent le même stockage et donc une seule d'entre elles peut être
+     définie par manipulation.
```
Comment (devnexen) [inline reference/curl/constants_curl_setopt.xml:18]: I think `handle` meant `curl handle` thus ought to be `gestionnaire`.

**Comment by devnexen** [comment]
will continue a bit later.

File: reference/curl/constants_curl_setopt.xml [hunk 3]
```diff
@@ -0,0 +1,3094 @@
...
+   <term>
+    <constant>CURLOPT_CONNECTTIMEOUT_MS</constant>
+    (<type>int</type>)
+   </term>
+   <listitem>
+    <para>
+     Le nombre de millisecondes à attendre lors de la tentative de connexion. Utilisez 0 pour
+     attendre indéfiniment.
+     <!-- http://curl.haxx.se/libcurl/c/curl_easy_setopt.html -->
+     Si libcurl est construit pour utiliser le résolveur de noms système standard, cette
```
Comment (devnexen) [inline reference/curl/constants_curl_setopt.xml:190]: I would prefer `compilé` over `construit`, I think there are couple of instances.

File: reference/curl/constants_curl_setopt.xml [hunk 4]
```diff
@@ -0,0 +1,3094 @@
...
+  </varlistentry>
+  <varlistentry xml:id="constant.curlopt-dns-use-global-cache">
+   <term>
+    <constant>CURLOPT_DNS_USE_GLOBAL_CACHE</constant>
+    (<type>int</type>)
+   </term>
+   <listitem>
+    <para>
+     &true; pour utiliser un cache DNS global. Cette option n'est pas thread-safe.
+     Elle est activée par défaut si PHP est construit pour une utilisation non threadée
```
Comment (devnexen) [inline reference/curl/constants_curl_setopt.xml:439]: As mentioned, there are other instances, a find/replace should do it :) anyhow take your time I ll do a last review most likely this evening.

File: reference/curl/constants_curl_setopt.xml [hunk 5]
```diff
@@ -0,0 +1,3094 @@
...
+   <term>
+    <constant>CURLOPT_PROXY_SSLKEY</constant>
+    (<type>int</type>)
+   </term>
+   <listitem>
+    <para>
+     Le nom de fichier de votre clé privée utilisée pour se connecter au proxy HTTPS.
+     Le format par défaut est "PEM" et peut être changé avec
+     <constant>CURLOPT_PROXY_SSLKEYTYPE</constant>.
+     (iOS et Mac OS X uniquement) Cette option est ignorée si curl a été compilé contre Secure Transport. Disponible si compilé avec TLS.
```
Comment (devnexen) [inline reference/curl/constants_curl_setopt.xml:1808]: I think `compilé avec` makes more sense than `compilé contre` wdyt ?

**Comment by devnexen** [comment]
The rest looks ok to me.

File: reference/curl/constants_curl_setopt.xml (hunk 5 above)
Comment (Fan2Shrek) [reply reference/curl/constants_curl_setopt.xml:1808]: Oops, I don't know what I had in mind x)

**Review by devnexen — APPROVED** [review]

**Pull request merged**